templates/message.mako

```diff
@@ -51,12 +53,11 @@
 
 ## Render large message.
 <%def name="render_large_message( message, status )">
-    <div class="${status}messagelarge" style="margin: 1em">${_(message)}</div>
+    <div class="${bleach.clean(status)}messagelarge" style="margin: 1em">${_(bleach.clean(message))}</div>
```

Comment:
[Bleach documentation](https://bleach.readthedocs.io/en/latest/clean.html) says it is not safe to use `clean()` for HTML attributes, which is why I implemented the previous validation approach.

Reply:
thanks, I've pushed a rebase